app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/home/HomeFragment.kt

```diff
@@ -2,17 +2,17 @@ package com.github.alfabravo2013.readyforexams.presentation.home
 
 import android.os.Bundle
 import android.view.View
-import androidx.fragment.app.viewModels
 import com.github.alfabravo2013.readyforexams.R
 import com.github.alfabravo2013.readyforexams.databinding.FragmentHomeBinding
 import com.github.alfabravo2013.readyforexams.presentation.BaseFragment
+import org.koin.android.ext.android.inject
 
 class HomeFragment : BaseFragment(R.layout.fragment_home) {
-    private val viewModel: HomeViewModel by viewModels()
+    private val viewModel: HomeViewModel by inject()
```

Comment:
You need to use by viewModel() to make the viewModel lifecycle aware. https://insert-koin.io/docs/reference/koin-android/viewmodel . Check this docs please.